include/api/wasmedge.h.in

```diff
@@ -35,6 +35,88 @@
 #define WASMEDGE_VERSION_MINOR ${WASMEDGE_VERSION_MINOR}
 #define WASMEDGE_VERSION_PATCH ${WASMEDGE_VERSION_PATCH}
 
+/// Error code enumeration.
+enum WasmEdge_ErrCode {
```

Comment:
I'm not entirely sure if these were intended to be omitted. I think they'd be useful for bindings users to recover from errors.

Reply:
I agree with you, errors are important by clear error codes and error message, may we make this better?